_posts/2022-11-02-gsoc2022-final.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "GSoC / Outreachy 2022 wrap-up"
+---
+
+<p>
+<img
+src="{{ site.baseurl }}{{ site.images }}/mdanalysis-gsoc.png"
+title="Google Summer of Code / Outreachy 2022" alt="Google Summer of Code / Outreachy 2022"
+style="float: right; height: 12em; " />
+</p>
+
+This year MDAnalysis participated in two student programs with wonderful students from around the globe. 
+We continued our participation in [Google Summer of Code][gsoc] for another year and also participated in [Outreachy][outreachy]! for the first time. 
+Huge thanks to these organisations for supporting MDAnalysis and the students.
+
+## Google Summer of Code
+ 
+We had[two GSOC projects][gsoc-projects] this year who made wonderful contributions.  None of this would have been possible without our students: [Aya Mohamed Alaa][project-aya] (@aya9aladdin), and [Bjarne Feddersen][project-bjarne] (@BFedder). 
+
+You can read more about their projects and packages in their blog posts:
+
+* Aya :  
+* Bjarne: 
+
+—  GSOC Mentors @jbarnoud @hmacdope @ojeda-e @IAlibay @fiona-naughton @orbeckst @lilyminium @richardjgowers
+
+## Outreachy 
+
+** Outreachy People fill in the details **
```

Comment:
@jbarnoud @micaela-matta @richardjgowers please add something short about Outreachy

Reply:
ping @jbarnoud @micaela-matta @richardjgowers — we need some prose in the blog post